test/plugin/base_test.rb

```diff
@@ -1089,15 +1089,15 @@ def test_http_request_skip_setting_latency
   end
 
   def test_log_entry_trace_field
-    verify_log_entry_field_key('trace', DEFAULT_TRACE_KEY,
-                               'custom_trace_key',
-                               CONFIG_CUSTOM_TRACE_KEY_SPECIFIED)
+    sample_value = 'projects/project-1/traces/1234567890abcdef1234567890abcdef'
+    verify_field_key('trace', DEFAULT_TRACE_KEY, 'custom_trace_key',
+                     CONFIG_CUSTOM_TRACE_KEY_SPECIFIED, sample_value)
```

Comment:
[Optional] I'd just inline the sample value into the call...

Reply:
Done here and for span ID also.